### PR TITLE
Add missing translations

### LINKED ISSUE #2877: Missing translations
I looked around the app in another language and spotted some parts that weren't translated. Double check that this text is all in resource files. If it already is then it might be new, and the translation isn't provided yet.

Level on structured logs:

image

### PR DESCRIPTION
**Pull request by adamint**
Add missing translations for structured logs page's "Level:" and the contains and not contains strings.

Fixes #2877 
 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/dotnet/aspire/pull/3408)

### CODE REVIEW

File: src/Aspire.Dashboard/Resources/Logs.resx [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+		<value>System.Resources.ResXResourceReader, System.Windows.Forms, Version=2.0.3500.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+	</resheader>
+	<resheader name="writer">
+		<value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=2.0.3500.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+	</resheader>
+  <data name="LogContains" xml:space="preserve">
+    <value>contains</value>
+  </data>
+  <data name="LogNotContains" xml:space="preserve">
+    <value>not contains</value>
```
Comment (danmoseley) [inline src/Aspire.Dashboard/Resources/Logs.resx:105]: just curious, is a fragment like this localizable (at least without more context)? I understand translating complete sentences, and maybe headings like "Level:" but in past projects localizer folks have asked me context here in comments for localizers explaining whether this stands alone, or needs to grammatically agree with something, or corresponds to singular or plural things...

IDK, just curious

Comment (adamint) [reply src/Aspire.Dashboard/Resources/Logs.resx:105]: Yes, it is localizable 😄 see the comment right below

**Review by kvenkatrajan — APPROVED** [review]

**Comment by danmoseley** [comment]
Should this be in release/8.0? for example
```
            FilterCondition.Contains => "contains",    // release 8.0
            FilterCondition.Contains => loc?[nameof(Logs.LogContains)] ?? "contains", //main
```

**Comment by danmoseley** [comment]
@JamesNK can you please figure out whether a change is missing here, and if so, prep a PR? 

cherry picking [6b9db81](https://github.com/dotnet/aspire/commit/6b9db8198ea14e723eea38cf057f43724ae0ab22) into release seems to produce changes we may want (?)

**Pull request merged**